beacon_node/beacon_chain/src/attestation_verification.rs

```diff
@@ -817,8 +817,9 @@ impl<'a, T: BeaconChainTypes> IndexedUnaggregatedAttestation<'a, T> {
         subnet_id: Option<SubnetId>,
         chain: &BeaconChain<T>,
     ) -> Result<(u64, SubnetId), Error> {
-        let expected_subnet_id = SubnetId::compute_subnet_for_attestation_data::<T::EthSpec>(
-            indexed_attestation.data(),
+        // TODO(electra) if we want
+        let expected_subnet_id = SubnetId::compute_subnet_for_attestation::<T::EthSpec>(
+            &attestation,
```

Comment:
the indexed_attestaton doesnt have committee_bits as a field, so we use the provided `AttestationRef` instead. Is that ok?